libpetrifui/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ file (GLOB LIBPETRIFUI_SOURCES *.c)
 include_directories (
     ${Petri-Foo_SOURCE_DIR}/libpetrifoo
     ${LIBXML2_INCLUDE_DIR}
-    ${OPENSSL_INCLUDE_DIR}
+    ${GCRYPTG_CFLAGS}
```

Comment:
This should probably be `${GCRYPT_INCLUDE_DIR}`. Will try that later and commit.